### ISSUE TITLE
Create Windows batch script for starting/stopping the bot

### ISSUE THREAD

**Issue by gazbert** [gazbert/BX-bot#9]
Create a Windows .bat file to start/stop the bot. Base it off the current linux bxbot.sh

**Comment by broomyocymru** [comment]
Alternatively you thought about running as a windows service e.g. https://nssm.cc/usage. Start|Stop|Status would then be trivial batch commands

**Comment by gazbert** [comment]
Yup, good idea. I use a linux service for the bot on my servers, but be tidy to have windows equivalent.  Got for it ;-)

P.S. There's a shed load of stuff going back onto master this weekend (REST API and the splitting of the bot into modules) from bxbot-restapi branch - so might be worth forking after that to save some pain...  will be gradle 'ing it this weekend too :-)
